notebooks/013_manage_models.ipynb

```diff
@@ -0,0 +1,403 @@
+    "    print(f\"Task {i + 1}: \", model.name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Optimizing a Model\n",
+    "If we want to increase the model's throughput or try out different precision levels, we can optimize the model using [OpenVINO Post Training Optimization feature](https://docs.openvino.ai/2024/openvino-workflow/model-optimization-guide/quantizing-models-post-training.html).\\\n",
+    "We will use the Model Client to start an optimization job."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job = model_client.optimize_model(model=active_models[0], optimization_type=\"pot\")\n",
+    "_ = model_client.monitor_job(job)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Work with model groups\n",
+    "Model Group is another entity in GETi that embraces all the trained models for a specific algorithm and task, differentiating them by their incrementing versions. Working with model groups allows to access different model versions and switch between them if needed, seting them active at the Platform server.\n",
+    "\n",
+    "The Model Client allows exploring model groups that are present in a project. If a model for some algorithm was trained for any task, the corresponding model group will exist in the project.\\\n",
+    "\n",
+    "> Note: you can reuse a model object from any of the previous steps, the model is retrieved using the Model Group further only for demonstration purposes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_groups = model_client.get_all_model_groups()\n",
+    "print(\"Model groups:\")\n",
+    "for i, model_group in enumerate(model_groups):\n",
+    "    print(f\"{i + 1}. \", model_group.name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Model Group actually operates models in a shortened form to increase SDK performance and lower server load, thus to retrieve the full model we need to make an additional call to Model Client.\\\n",
+    "Now we can restore a representation object for a specific model from the model group."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_summary = model_groups[1].models[0]\n",
+    "model = model_client.update_model_detail(model_summary)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Remember we performed an optimization job for the model? Now we can also retrieve the optimized model from the model object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "optimized_model = model.get_optimized_model(optimization_type=\"pot\")\n",
```

Comment:
In addition, there is the `ModelClient.get_latest_optimized_model()` method that allows filtering by algo name, optimization_type, precision and xai capabilities.
Might be good to show that one as well, as an alternative approach.

Reply:
done!